Packs/Box/Integrations/README.md

```diff
@@ -0,0 +1,535 @@
+Manage Box users
+## Configure Box on Cortex XSOAR
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Box.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| insecure | Trust any certificate \(not secure\) | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+## How to initialise:
```

Comment:
@kirbles19 @ShirleyDenkberg please review this section

Reply:
Section's header took from https://github.com/demisto/etc/issues/14446